doc/source/examples/tools_example.rst

```diff
@@ -40,16 +41,17 @@ The function will return
 
 .. code-block:: python
 
-        {"email": "janedoe@email.com", "username": "Jane Doe"}
+        {"owner_email": "janedoe@email.com", "owner_name": "Jane Doe", "owner_orcid": "0000-0000-0000-0000"}
 
 
 Where does ``get_user_info()`` get the user information from?
 -------------------------------------------------------------
 
 The function will first attempt to load the information from configuration files with the name ``diffpyconfig.json``
 on your hard-drive.
-It looks first for the file in the current working directory.  If it cannot find it there it will look
-user's home, i.e., login, directory.  To find this directory, open a terminal and a unix or mac system type ::
+It looks for files in the current working directory and in the computer-user's home (i.e., login) directory.
+For example, it might be in C:/Users/yourname`` or something like that, but to find this directory, open
```

Comment:
much clearer since there is an example provided since i wasn't sure what "logic" meant here.